ci/release-qualification/pipeline.template.yml

```diff
@@ -164,6 +164,7 @@ steps:
 
   - id: tests-preflight-check-rollback
     label: Tests with preflight check and rollback
+    skip: "TODO(def-): Too complex to adapt for all mzcompose-based tests"
```

Comment:
The use of a TODO means that this will change?

Reply:
If I find time.